.storybook/themes/news.scss

```diff
@@ -1799,16 +1799,17 @@
 					margin: 0 0 var(--global-spacing-5) 0,
 				),
 				right-rail-full-width-1: (
-					margin: 0 0 var(--global-spacing-5) 0,
+					margin: 0 var(--global-spacing-4) var(--global-spacing-5),
+					// margin: 0 0 var(--global-spacing-5) 0,
 				),
 				right-rail-main: (
 					gap: var(--global-spacing-6),
 					max-width: calc(var(--content-max-width) * 1px),
-					width: var(--content-scale-width),
+					width: var(--layout-max-width),
 					margin: auto,
 				),
 				right-rail-full-width-2: (
-					margin: var(--global-spacing-6) 0 0 0,
+					margin: var(--global-spacing-6) var(--global-spacing-5) 0,
```

Comment:
We don't want to set margins here since the blocks will control their own width with `content-scale-width`